src/api-client.ts

```diff
@@ -14,20 +14,48 @@ const enum TranscriptContentTypes {
     TEXT = 'text/plain'
 }
 
+/**
+ * Client which handles connection to the rev.ai API.
+ */
 export class RevAiApiClient {
     apiHandler: ApiRequestHandler;
+
+    /**
+     * @param accessToken Access token used to validate API requests
+     * @param version (optional) version of the API to be used
+     */
     constructor (accessToken: string, version = 'v1') {
         this.apiHandler = new ApiRequestHandler(`https://api.rev.ai/revspeech/${version}/`, accessToken);
     }
 
+    /**
+     * See https://www.rev.ai/docs#tag/Account
+     * Get information associated with the account whose access token is used by this client
+     * @returns Account object
+     */
     async getAccount(): Promise<RevAiAccount> {
         return await this.apiHandler.makeApiRequest<RevAiAccount>('get', '/account', {}, 'json');
     }
 
+    /**
+     * See https://www.rev.ai/docs#operation/GetJobById
+     * Get information about a specific job
+     * @param id Id of job whose details are to be retrieved
+     * @returns Job details
+     */
     async getJobDetails(id: string): Promise<RevAiApiJob> {
         return await this.apiHandler.makeApiRequest<RevAiApiJob>( 'get', `/jobs/${id}`, {}, 'json');
     }
 
+    /**
+     * See https://www.rev.ai/docs#operation/GetListOfJobs
+     * Get a list of transcription jobs submitted within the last week in reverse chronological order
```

Comment:
maybe put `(Last submitted first)`, just to clarify what reverse chronological order means